src/OpenTelemetry.Extensions.EventSource/README.md

```diff
@@ -0,0 +1,38 @@
+# OpenTelemetry.Extensions.EventSource
+
+[![NuGet](https://img.shields.io/nuget/v/OpenTelemetry.Extensions.EventSource.svg)](https://www.nuget.org/packages/OpenTelemetry.Extensions.EventSource)
+[![NuGet](https://img.shields.io/nuget/dt/OpenTelemetry.Extensions.EventSource.svg)](https://www.nuget.org/packages/OpenTelemetry.Extensions.EventSource)
+
+This project contains an
+[EventListener](https://docs.microsoft.com/dotnet/api/system.diagnostics.tracing.eventlistener)
+which can be used to translate events written to an
+[EventSource](https://docs.microsoft.com/dotnet/api/system.diagnostics.tracing.eventsource)
+into OpenTelemetry logs.
+
+## Installation
+
+```shell
+dotnet add package OpenTelemetry.Extensions.EventSource
```

Comment:
minor thing here.. but it's occurred to me that we have this line in all of our readmes and it doesn't work for many of our packages which do not yet have a stable release without using the `--prerelease` switch.

```suggestion
dotnet add package OpenTelemetry.Extensions.EventSource --prerelease
```

Reply:
Updated here and in serilog project README.